chucks-pick3-web/angular-app/src/app/services/member.service/member.service.ts

```diff
@@ -10,10 +10,12 @@ export class MemberService {
 
   }
 
-  /* Add Create Account Here*/
+
   public createAccount(): boolean {
-    console.log('memberServiceTest');
     return false;
   }
+  public logInSuccessful() {
```

Comment:
Need to rename to a better more intuitive name that accurately reflects what the user is doing when the click on the login submit button.

Reply:
renamed to performLogIn